README.md

```diff
@@ -1,6 +1,10 @@
 ## Calico Network Policy for Kubernetes 
 
-This repository contains the Calico Kubernetes policy controller, which implements the Kubernetes network policy API.  The controller uses the Kubernetes v1beta1 network policy API to configure Calico network policy.  The policy controller addon is deployed on top of Kubernetes as a set of replicated pods, and can also be installed as a static pod. 
+This repository contains the Calico Kubernetes policy controller, which implements the Kubernetes network policy API.  
+
+![calico-policy-controller](calico-policy-controller.png)
```

Comment:
The image isn't quite right.

The policy controller watches the following k8s resources:
- Pods
- Namespaces
- NetworkPolicies

What was the motivation for this README change? Do we really need a more in-depth page on "how it works", as discussed in https://github.com/projectcalico/k8s-policy/issues/30